tests/comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -991,6 +991,51 @@ wrap(
 ================================================================================
 `;
 
+exports[`emoji.js 1`] = `
+====================================options=====================================
+parsers: ["flow", "babel", "typescript"]
+printWidth: 80
+                                                                                | printWidth
+=====================================input======================================
+/* #2091 */
+
+const test = '💖'
+// This comment
+// should not get collapsed
+
+=====================================output=====================================
+/* #2091 */
+
+const test = "💖";
+// This comment
+// should not get collapsed
+
+================================================================================
+`;
+
+exports[`emoji.js 2`] = `
+====================================options=====================================
+parsers: ["flow", "babel", "typescript"]
+printWidth: 80
+semi: false
+                                                                                | printWidth
+=====================================input======================================
+/* #2091 */
+
+const test = '💖'
+// This comment
+// should not get collapsed
+
+=====================================output=====================================
+/* #2091 */
+
+const test = "💖"
```

Comment:
Did we missed `;` here ??

Reply:
Strange, [playground](https://deploy-preview-7368--prettier.netlify.com/playground/#N4Igxg9gdgLgprEAuEB6AVAAgMQCYAMAnAIybqoA6UVkUAzjJvA5gLyYDkgvBuBpOx1alSYAKgAsAlnUyQAtjIQwBQuqIgBXADYATTFAiMA5nEaQNGgIYAHOnC1UQAGhARLMcdDrJQ5gE4+IAO4ACr4InijmAG4Q4lqOIABGPuZgANbGAMqWKeJQBsgwPmpwTqIwMhoA6hLM2WBwGWHibpHNAJ7I4HSeTrk2PjBByQYy5sgAZuYaNk4AVnQAHgBCyWmZ5vIAMrlwE1MzIPMLGbkGGnAAimr6u0iT0yUg2T79nTBtlnB0YD7irvGWX6wSqxGCiZAADnwTkBEBslWSlk6gK+cB8kV2TgAjtd4EMXOEQOY6ABaKBwWy2eI+OA48Q0obmEZjO77R42GTiApFdmnc5XG57B5OGDmBIgrRg5C4EXJcQaU4AYQgchZID05PiahswjF4XuB0ixQAklAtAoMj8-jAAIJmjLvc5CmwAXxdQA) is good

Reply:
Sorry, didn't see `semi: false`